src/commands/helm-lint.yaml

```diff
@@ -1,8 +0,0 @@
----
-parameters:
```

Comment:
Do we use this command somewhere in our projects currently? If so it would be good to have a list of repos where we use it and make sure we transition them. Otherwise someone will run into this when trying to update the orb.

Reply:
I know just about one, kong-app, but will announce it anyway